README.md

```diff
@@ -1,45 +1,99 @@
+# Sherlock Platform
+
+## About
+
+This is the customized IntelliJ platform for Sherlock Project - an IntelliJ-based implementation of the new GPU Profiler tool. This is based
+off the IntelliJ Platform Version 2.1. To read more on the IntelliJ Platform documentation scroll down to the IntelliJ IDEA Community
```

Comment:
IntelliJ Platform version `2024.2`.

Reply:
It still says `2.2`, it should say `2024.2` I believe.

Reply:
Sorry, missed it. Updated now.